https://github.com/PyO3/maturin/commit/fe41da6a7d1ad5940bfec161ed1d2d556d0856fb
but platforms create doesn't know of *-alpine-linux-musl yet
--- a/src/target.rs
+++ b/src/target.rs
@@ -366,6 +366,7 @@ fn get_host_target() -> Result<String> {
                 output
             )
         })?
+        .replace("alpine", "unknown")
         .to_string();
     Ok(host)
 }
